contents/tutorindigo.rst

```diff
@@ -0,0 +1,245 @@
+        font-weight: 700;
+        font-style: normal;
+    }
+
+    @font-face {
+        font-family: 'Crimson';
+        src: url('#{$static-path}/fonts/CrimsonText-BoldItalic.ttf') format('truetype');
+        font-weight: 700;
+        font-style: italic;
+    }
+
+Here, you have to ensure that ``font-weight`` and ``font-style`` should match with the respective files. Like, ``fonts/CrimsonText-Bold.ttf`` has font-weight 700 and font-style normal.
+
+- If you don't want to add font files and use Google font URL. Then, add the below url in the ``tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_fonts.scss`` file (for `Crimson font family <https://fonts.google.com/specimen/Crimson+Text>`_)::
+
+    @import url('https://fonts.googleapis.com/css2?family=Crimson+Text:ital,wght@0,400;0,600;0,700;1,400;1,600;1,700&display=swap');
+
+- After that, In ``tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_extras.scss`` file, add the below line for loading the fonts in LMS platform::
+
+    @import "fonts"; 
+
+- Lastly, Set font family in ``tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_variables.scss`` file as below::
+
+    $font-family-title: 'Crimson', sans-serif;;
+    $serif: Crimson;
+    $f-sans-serif: Crimson, 'Open Sans','Helvetica Neue', Helvetica, Arial, sans-serif;
+
+The above steps need to be followed for CMS. But CMS requires one more step in addition to above steps for changing font family.
+
+- Include the below line in ``tutor-indigo/tutorindigo/templates/indigo/cms/static/sass/partials/cms/theme/_variables-v1.scss`` file after ``$static-path`` definition::
+
+    $baseline: 20px;
+
+    $static-path: '..' !default;
+
+    @import "fonts";   /* add this line */
+
+    ......
+
+After doing so, Run ``tutor config save`` &  ``tutor local stop``. Run ``tutor images build openedx`` to build the Open edX image with the updated images. Finally, start the platform using ``tutor local start -d``. You can successfully view the updated font family using browser inspection tool as in the below image.
+
+.. list-table:: Font Updations Example
+   :widths: 50 50
+   :header-rows: 1
+
+    * - .. image:: /academy/static/images/tutorindigo/lms_font_change.png
+            :width: 400
+            :alt: LMS Inter Font Style Example
+      - .. image:: /academy/static/images/tutorindigo/cms_font_change.png
+            :width: 400
+            :alt: CMS Inter Font Style Example
+    
+
+Sass Styles Adjustments
+-----------------------
+
+Customize styles in ``tutorindigo/templates/indigo/lms/static/sass`` for LMS and ``tutorindigo/templates/indigo/cms/static/sass`` for CMS. Note that ``tutorindigo/templates/indigo/lms/static/sass/partials/lms/theme/_extras.scss`` file is used for adding and overriding styles. For Example, To change the background-color of body, follow the below steps:
+
+- Clone the Tutor Indigo by running the command ``git clone https://github.com/overhangio/tutor-indigo.git``
```

Comment:
Let's get rid of the bullet points.

Clone the Tutor Indigo repository by running:

    git clone https://github.com/overhangio/tutor-indigo.git


By the way, shouldn't these instructions also be followed for the "Font Modifications" and other sections above? We are instructing users to git clone the Indigo repo in four different places. I suggest we move these instructions to a top-level "How to make changes to the Indigo plugin" subsection.